.appveyor.yml

```diff
@@ -42,7 +42,8 @@ install:
 
     # Configure the VM.
     - cmd: conda.exe install -n root --quiet --yes conda-forge-ci-setup=2
-    - cmd: run_conda_forge_build_setup
+    # Overriding global run_conda_forge_build_setup_win with local copy.
```

Comment:
Why?

Reply:
https://github.com/conda-forge/conda-smithy/issues/931

And we need to get a dependency from a custom label here.

Reply:
@mariusvniekerk do you think we can merge this? If it's ok, I would prefer not to wait for a new release of conda-smithy. We will update the feedstock accordingly when the release happens.